src/MoBi.Presentation/Presenter/CreateSimulationConfigurationPresenter.cs

```diff
@@ -100,10 +100,20 @@ private void updateSimulationConfiguration(SimulationConfiguration simulationCon
          var selectedExpressions = individualAndExpressionPresenter.ExpressionProfiles;
          var selectedIndividual = individualAndExpressionPresenter.SelectedIndividual;
 
-         _simulationConfigurationTask.UpdateFrom(simulationConfiguration, moduleConfigurations, selectedIndividual, selectedExpressions);
+         updateFrom(simulationConfiguration, moduleConfigurations, selectedIndividual, selectedExpressions);
```

Comment:
Personally, i find this code less readable. I do not. know of this is a copy or a clone happening, , the method is only used once and 4 parameters are passed. 4 lines of method declarations . Can't say that I appreciate the benefits over the costs.